### PR TITLE
Release branch with all feautures added

### PR DESCRIPTION
**Pull request by SantiagoGuerra**

### CODE REVIEW

**Comment by Ispirett** [comment]
# Reviewing (TSE )(IspireTT)

File: index.html
Comment (Ispirett) [inline index.html]: You should use HTML and CSS here instead of images like the original site.

**Review by Ispirett — CHANGES_REQUESTED** [review]
Hey, @SantiagoGuerra @Vashiramusa  Really good work with this project so far.
image. 

There are just a few things to take care of.

### Sign_in button
Ensure there is a Sign In button at the top right corner. :+1: 

### Footer
* [ ] It contains copyright and address information.

### Readme File
Please add some information about the project and the contributor's names. 

### Stickler 
For future html projects you can use this setup https://github.com/microverseinc/linters-config/tree/master/css

File: .stickler.yml
Comment (bolah2009) [inline .stickler.yml]: #### `REQUIRED`

Add the appropriate `.stickler.yml` file to the project. Let's use [this file](https://github.com/microverseinc/linters-config/blob/master/css/.stickler.yml) instead. :point_down: 

``` yml
linters:
  stylelint:
    config: 'stylelint.config.js'
```
- Also, add the `stylelint.config.js` that was referenced above, this can be found [here](https://github.com/microverseinc/linters-config/blob/master/css/stylelint.config.js)

If you are having an issue with setting up linter, read the `Troubleshooting` section of [this repo](https://github.com/microverseinc/linters-config/tree/master/css).

File: README.md
Comment (bolah2009) [inline README.md]: #### `HINT`

Add a descriptive readme. Details like assignment link (from Odin), live link (from [githack](https://raw.githack.com/)), screenshot and what was learned during the project can make the project more descriptive.

File: index.html
Comment (bolah2009) [inline index.html]: #### `HINT`

:bulb: Consider adding a `footer` element instead of as this is clearly the footer of this section. This gives the page more semantic meaning.

```suggestion
        <footer class="footer">
```

File: index.html
Comment (bolah2009) [inline index.html]: #### `HINT`

:bulb: Consider adding a `footer` element instead of as this is clearly the footer of the page. This gives the page more semantic meaning.

```suggestion
    <footer class="bottom-footer">
```
Learn more about footer [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/footer)

File: index.html
Comment (bolah2009) [inline index.html]: #### `HINT` 

:bulb: Using `main` here instead shows the dominant content of the `<body>` of a document. 
```suggestion
    <main class="main-content">
```

More on `<main>` [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main) :star2:

**Review by bolah2009 — CHANGES_REQUESTED** [review]
## Nice Job Team  :clap: Almost there :smile: 

The [clone](https://raw.githack.com/Vashiramusa/HTML-FORMS/release/index.html) looks nice, use HTML forms effectively. 

Before giving a :+1: let's make some **required** changes.
- Use the appropriate linter file instead
- Use semantic elements. Think of where semantic elements would have made the page more accessible.
   > Remember to use [semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp).
- Add a descriptive readme
   > Remember about descriptive README file.
- [Next Project] Use GitHub Flow: For HTML & CSS projects, use only one feature branch (e.g. `embedding-images-and-video`, `html-forms`, `responsive-design`, etc) and **do not merge any feature branch to master until it's been approved**. (`release` is not descriptive for the whole feature)
   > Follow the [Github Flow guidelines](https://microverse.pathwright.com/library/microverse-onboarding/86777/path/step/53987922/).

Kindly go through the [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59524927/) and [odin](https://www.theodinproject.com/courses/html5-and-css3/lessons/html-forms) project requirements for more details.

Good luck with your next submission. :star2: 

Cheers
[Bola](https://bolabuari.com/)

###### PS: Note that comments tagged as `OPTIONAL` are not part of the minimum requirements to approve this PR

**Review by stratospherique — COMMENTED** [review]
Great Work guys :clap: :clap: .

**Review by stratospherique — APPROVED** [review]
Great Work guys :clap: :clap: . You took care of everything and your page looks amazing. :clinking_glasses: :clinking_glasses: . You deserve the approval to merge to master and here is your thumbs up :+1: :+1: :100: :100: .
[OPTIONAL] Add a live preview of your page to your README file to showcase what you have accomplished.
 
# Go ahead merge and move on to the next project.

**Pull request merged**
